app/src/main/java/at/roteskreuz/stopcorona/model/entities/configuration/ApiConfiguration.kt

```diff
@@ -29,10 +30,12 @@ data class ApiConfiguration(
     val diagnosticQuestionnaire: ApiDiagnosticQuestionnaire?,
     @field:Json(name = "page_list")
     val pageList: ApiPageList?,
-    @field:Json(name = "upload_keys_days")
     // TODO mihbat 12-Jun: The constant can be removed when the backend will start providing
     //  this value.
-    val uploadKeysDays: Int = Constants.ConfigurationDefaults.UPLOAD_KEYS_DAYS
+    @field:Json(name = "upload_keys_days")
+    val uploadKeysDays: Int = Constants.ConfigurationDefaults.UPLOAD_KEYS_DAYS,
+    @field:Json(name = "exposure_configuration")
+    val exposureConfiguration: ApiExposureConfiguration
 ) : ApiEntity<DbConfiguration> {
 
     override fun asDbEntity(): DbConfiguration {
```

Comment:
You've forgotten to pass all api values to the `DbConfiguration` instance.

Reply:
@falkorichter

Reply:
`exposureConfigurationMinimumRiskScore = exposureConfiguration?.minimumRiskScore`